include/dpp/guild.h

```diff
@@ -259,6 +260,14 @@ class DPP_EXPORT guild_member {
 	 */
 	guild_member& set_nickname(const std::string& nick);
 
+	/**
+	 * @brief Get the dpp::user object for this member
+	 * @return dpp::user user object
```

Comment:
needs to indicate that if the user isn't in cache it returns nullptr

Reply:
Added